Assignment1/assignment1/lib/Types.dart

```diff
@@ -0,0 +1,38 @@
+class Type {
+  double calculateTax(double price) {
+    return 0.0;
+  }
+
+  double getTotal(double tax, double price, int quanitity) {
+    return (tax + price) * quanitity;
+  }
+}
+
+class Raw extends Type {
+  @override
+  double calculateTax(double price) {
+    return price * 0.125;
+  }
+}
+
+class Manufactured extends Type {
+  @override
+  double calculateTax(double price) {
+    return price * 0.125 + (price * 0.125 + price) * 0.02;
+  }
+}
+
+class Imported extends Type {
+  @override
+  double calculateTax(double price) {
+    double tax = price * 0.1;
```

Comment:
Extract the constant values like 0.1, 5, 10 etc. into constants and name them appropriately so that it is easier to understand what a particular value represents. For e.g., 0.1 can be named as IMPORT_DUTY_PERCENTAGE.

Reply:
static const IMPORT_DUTY_PERCENTAGE = 0.1;

Reply:
done